motoman_mh110_support/urdf/mh110_macro.xacro

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" ?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+  <xacro:macro name="motoman_mh110" params="prefix">
+    <xacro:include filename="$(find motoman_resources)/urdf/common_materials.xacro"/>
+
+    <!-- link list -->
+    <link name="${prefix}base_link">
+      <visual>
+        <geometry>
+          <mesh filename="package://motoman_gp110_support/meshes/gp110/visual/base_link.stl"/>
```

Comment:
Could you please add a comment here explaining why the MH110 package is using files from the GP110 package?

Just one here should suffice. No need to repeat it on the other `mesh` elements.